.github/workflows/master-to-im-master.yml

```diff
@@ -0,0 +1,53 @@
+name: Merge master to im-master PR
+
+on:
+  push:
+    branches:
+      master
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  REPO_NAME: ${{ github.repository }}
+  SOURCE_BRANCH: master
+  TARGET_BRANCH: im-master
+  PR_TITLE: "Automated Pull Request"
+  PR_BODY: "Merge master to im-master"
+  REVIEWERS: '["dshamaev-intermedia", "anikitin-intermedia", "esmitenko-intermedia", "inekrasov-intermedia"]'
```

Comment:
Maybe we should fetch this list from ci properties too?

Reply:
idk
We can use a CODEOWNERS file, but it would work for any pull request. I haven't found a way to get users using the GitHub context. One option is to use the GitHub API, but it seems like an added complexity.